src/Core/Resolvers/MsSqlDbExceptionParser.cs

```diff
@@ -85,5 +86,11 @@ public override bool IsTransientException(DbException e)
             string errorNumber = ((SqlException)e).Number.ToString();
             return TransientExceptionCodes.Contains(errorNumber);
         }
+
+        /// <inheritdoc />
+        public override DatabaseType DeriveDatabaseType()
+        {
+            return DatabaseType.MSSQL;
+        }
```

Comment:
When is this used? I only see references to this as overrides in all DB types and method signature in the abstract base class.

Reply:
This was a mistake. had needed it based on an earlier implementation and forgot to clear it once realized not needed. fixed now.